packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -31,6 +31,15 @@ export type OdysseyThemeProviderProps = {
   themeOverride?: ThemeOptions;
 };
 
+declare module "@mui/material/styles" {
+  interface Theme {
+    contrastMode: string;
+  }
+  interface ThemeOptions {
+    contrastMode?: string;
+  }
+}
```

Comment:
Is this required or can it be removed? Looks like we still have some styles in here doing `theme.contrastMode`.